### PR TITLE
add tests for clean_formatting and fix edge cases

### PR DESCRIPTION
**Pull request by mgoldey**
Testing now in place to ensure basic functionality works

### CODE REVIEW

File: asrtoolkit/clean_formatting.py [hunk 1]
```diff
@@ -60,17 +60,22 @@ def dollars_to_string(input_string):
...
   >>> dollars_to_string("$3.")
   'three dollars'
+  >>> dollars_to_string("$3.5")
+  'three dollars and fifty cents'
   """
   ret_str = input_string
   if contains_dollar_sign(input_string):
     input_string = input_string.replace("$", "")
-    ret_str = " dollars ".join(
+    dollars, cents = input_string.split(".")
```
Comment (ageojo) [inline asrtoolkit/clean_formatting.py]: will raise ValueError for non-decimal numeric types:
